torch/distributed/optim/functional_adamw.py

```diff
@@ -108,7 +108,7 @@ def step_param(self, param: Tensor, grad: Optional[Tensor]):
 
         state_steps.append(state["step"])
         with torch.no_grad():
-            F.adamw(
+            F.adam(
```

Comment:
It actually entirely broke for some reason, when it called F.adamw with jitscript it said "item adam is not defined" and I was so confused because just calling adam worked.

Reply:
Oh yeah I didn't realize  but you're right, doesn't this mean that F.adamw is just entirely broken with jitscript if this doesn't work? That doesn't seem great

Reply:
oh that's not good. we should def fix this before merging